src/js/utils/api.js

```diff
@@ -20,8 +27,10 @@ export default class api {
     return new Promise((resolve, reject) => {
       try {
         const data = JSON.parse(localStorage.getItem(NAMESPACE) || STRINGIFY_INITIAL_STORE);
+        const tableName = apiPathToTableName(requestPath);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).